src/components/common/FurnitureGallery/FurnitureGallery.module.scss

```diff
@@ -0,0 +1,160 @@
+@import "../../../styles/settings.scss";
+
+.root {
+  .productsRow{
+    justify-content: left;
+    .product {
+      flex-basis: 25%;
+      max-width: 25%;
+    }
+  }
+  .panelHeader {
+    margin-bottom: 30px;
+    position: relative;
+    border-bottom: 4px solid $global-border-color;
+    
+
+    .heading {
+      position: relative;
+
+      h3 {
+        color: $primary;
+        text-transform: uppercase;
+        font-size: 20px;
+        line-height: 38px;
+        margin: 0;
+        letter-spacing: 1px;
+      }
+    }
+  }
+
+  .productPhoto {
+    transform: translateY(35%);
+    height: 80%;
```

Comment:
Nie podoba mi się ani height ani transform, jest to strikte pod to zdjęcie dostosowane, a zdjęcia w sklepie będą się zmieniać, raczej to zdjęcie powinno być wrzucone tak przystosowane aby wyświetliło się poprawnie.

Reply:
productPhtoto  to div do zdjęcia, później na zdjęcu jest dodane cover, sprawdzałem to na kilku rozdzielczościach zdjęcia, powinno być teraz ok :) zerknij proszę na ostatnie zmiany :)